65.md

```diff
@@ -4,73 +4,56 @@ NIP-65
 
-  - Most people are sending their posts to the same most popular relays in order to be more widely seen
-  - Many people are pulling from a large number of relays (including many duplicate events) in order to get more data
+  - Most users submit their posts to the same highly popular relays, aiming to achieve greater visibility among a broader audience.
+  - Many users are pulling events from a large number of relays in order to get more data at the expense of duplication
   - Events are being copied between relays, oftentimes to many different relays
+  
+This NIP allows Clients to connect directly with the most up-to-date relay set from each individual user, eliminating the need of broadcasting events to popular relays. 
 
-### Purposes
-
-The purpose of this NIP is to help clients find the events of the people they follow, to help tagged events get to the people tagged, and to help nostr scale better.
-
-### Suggestions
+## Final Considerations
 
-It is suggested that people spread their kind `10002` events to many relays, but write their normal feed-related events to a much smaller number of relays (between 2 to 6 relays). It is suggested that clients offer a way for users to spread their kind `10002` events to many more relays than they normally post to.
+1. Clients should guide users to keep `kind:10002` lists small (2-4 relays). 
 
-Authors may post events outside of the feed that they wish their followers to follow by posting them to relays outside of those listed in their "Relay List Metadata".  For example, an author may want to reply to someone without all of their followers watching.
+2. Clients should spread an author's `kind:10002` events to as many relays as viable, all of them if possible. 
 
-It is suggested that relays allow any user to write their own kind `10002` event (optionally with AUTH to verify it is their own) even if they are not otherwise subscribed to the relay because
+3. `kind:10002` events should only be used to advertise relays. The user's configuration when using Clients should stay in Kind 3.
 
-  - finding where someone posts is rather important
-  - these events do not have `content` that needs management
-  - relays only need to store one replaceable event per pubkey to offer this service
-
-### Why not in kind `0` Metadata
-
-Even though this is user related metadata, it is a separate event from kind `0` in order to keep it small (as it should be widely spread) and to not have `content` that may require moderation by relay operators so that it is more acceptable to relays.
-
-### Example
-
-```json
-{
-  "kind": 10002,
-  "tags": [
-    ["r", "wss://alicerelay.example.com"],
-    ["r", "wss://brando-relay.com"],
-    ["r", "wss://expensive-relay.example2.com", "write"],
-    ["r", "wss://nostr-relay.example.com", "read"],
-  ],
-  "content": "",
-  ...other fields
-```
+4. DMs should be broadcasted to the author's WRITE relays and to the receiver's READ relays to keep maximum privacy. 
```

Comment:
This is fine for now.  I think I'll do a separate PR that adds a "dm" marker which is like "write" but is specific to DMs.  This PR can just be about clarification.

Reply:
Hang on I didn't read it right.  Why should DMs go to the authors WRITE relays?  So the author can read them back? Maybe.  In either case, this doesn't do anything that improves privacy, it is puttng the DMs everywhere you might put anything. So I wouldn't say "to keep maximum privacy."

Reply:
Is there any way we can generalize this to private events (like a private zap) and not only DMs?